rts-core/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+tokio = { version = "1.16.1", features = ["full"] } # @TODO remove
```

Comment:
Why isn't full needed? (or why was it needed?)

Reply:
I used Tokio to quickly test that async was working. I though that actix and tokio have differents runtime, so I put a todo to not forget to remove before merge. But we discussed, and it is the same runtime, so I guess it can stay